app/renderer/components/tabContent.js

```diff
@@ -223,7 +223,7 @@ class TabTitle extends ImmutableComponent {
   get themeColor () {
     const themeColor = this.props.tab.get('themeColor') || this.props.tab.get('computedThemeColor')
     const activeNonPrivateTab = !this.props.tab.get('isPrivate') && this.props.isActive
-    const activePrivateTab = this.props.tab.get('isPrivate') && this.props.isActive
+    const activePrivateTab = (this.props.tab.get('isPrivate') && this.props.isActive) || this.props.isHover
```

Comment:
we have a tab property for that, `this.props.tab.get('hoverState')` which is a prop made for hover events. I think that's the right fit for these changes and you can just remove other mentions for hover you created.

You could also change `const` name to update its meaning for something like `isPrivateTab` since we're adding a rule that's not exclusive of active  privateTabs. i.e.:

```
const activePrivateTab = this.props.tab.get('isPrivate') && (this.props.isActive || this.props.tab.get('hoverState'))
```